build.sbt

```diff
@@ -143,7 +143,7 @@ inThisBuild(
     githubWorkflowBuild +=
       WorkflowStep.Sbt(List(s"++$Scala213", "site/makeMicrosite"), cond = Some(Scala213Cond)),
     githubWorkflowPublish := Seq(
-      WorkflowStep.Sbt(List("release")),
+      WorkflowStep.Sbt(List("tlRelease")),
```

Comment:
Should this be a `+=` instead?

Reply:
~~Uh, probably.~~

Well, I don't know.  sbt-spiewak just sets it to `release`, overriding `publish`.  I guess it depends what `tlRelease` does, but it appears to already be an alias over publish.

Reply:
`tlRelease` is sbt-spiewak's `release` with some slight modification to do snapshots properly.

Reply:
You could also consider moving these steps to `githubWorkflowPublishPostamble`. But in general overriding the steps generated by sbt-typelevel should be (mostly) an anti-pattern.